RPI/csproject_co2_reading.sql

```diff
@@ -4,6 +4,7 @@ CREATE SEQUENCE IF NOT EXISTS csproject_co2_reading_id_seq;
 -- Table Definition
 CREATE TABLE "public"."csproject_co2_reading" (
     "id" int4 NOT NULL DEFAULT nextval('csproject_co2_reading_id_seq'::regclass),
+    "sensor_id" VARCHAR(12) NOT NULL,
```

Comment:
I use varchar here, so we might need to update the type in `csproject_sessions` (from int to varchar).